.github/workflows/dapr-automerge.yml

```diff
@@ -0,0 +1,26 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+jobs:
+  automerge:
+    if: github.repository_owner == 'dapr'
+    name: Automerge and update PRs.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Automerge and update
+        env:
+          MAINTAINERS: yaron2,AaronCrawfis,orizohar,msfussell
```

Comment:
Can this be the @dapr\maintainers-docs group instead of individual users?

Reply:
Well. That would be a follow up. The script does not do that now.

Reply:
@AaronCrawfis Is this a blocker to merge this PR?

Reply:
Nope, just behind on PRs 😄